lambdas/notify_old_deploys/notify_old_deploys.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python
+# -*- encoding: utf-8 -*-
+"""
+Checks the DynamoDB deployments table for deployments with a createdAt date
+older than the given AGE_BOUNDARY_MINS.
+
+Publishes a message to a given SNS topic if any out of date deployments are
+found.
+
+This lambda is intended to be run on a repeated schedule of some period less
+than AGE_BOUNDARY_MINS.
+"""
+
+from datetime import datetime, timedelta, timezone
+from functools import partial
+import os
+import pprint
+
+import boto3
+
+from deployment_utils import get_deployments_from_dynamo
+from sns_utils import publish_sns_message
+
+
+def _old_deployment(age_boundary_mins, deployment):
+    age_boundary = datetime.now(timezone.utc) - \
```

Comment:
Not a super fan of line continuation, personally. I think it's cleaner to wrap everything in parens, add a newline and indent. i.e.

```python
age_boundary = (
   datetime.now(timezone.utc) - timedelta(minutes=age_boundary_mins)
)

return (
    (deployment.created_at < age_boundary) and (deployment.color == 'green')
)
```